test/functional/rpc_createmultisig.py

```diff
@@ -164,17 +160,23 @@ def do_multisig(self):
         if self.output_type == 'bech32':
             assert madd[0:4] == "bcrt"  # actually a bech32 address
 
-        # compare against addmultisigaddress
-        msigw = wmulti.addmultisigaddress(self.nsigs, self.pub, None, self.output_type)
-        maddw = msigw["address"]
-        mredeemw = msigw["redeemScript"]
-        assert_equal(desc, drop_origins(msigw['descriptor']))
-        # addmultisigiaddress and createmultisig work the same
-        assert maddw == madd
-        assert mredeemw == mredeem
-
-        txid = node0.sendtoaddress(madd, 40)
-
+        if self.is_bdb_compiled():
+            # compare against addmultisigaddress
+            msigw = wmulti.addmultisigaddress(self.nsigs, self.pub, None, self.output_type)
+            maddw = msigw["address"]
+            mredeemw = msigw["redeemScript"]
+            assert_equal(desc, drop_origins(msigw['descriptor']))
+            # addmultisigiaddress and createmultisig work the same
+            assert maddw == madd
+            assert mredeemw == mredeem
+            wmulti.unloadwallet()
+
+        if self.output_type == "bech32":
+            spk = bytes.fromhex(node0.validateaddress(madd)["scriptPubKey"])
```

Comment:
I think you can remove the if-else and just unconditionally use `validateaddress` for simplicity.